### ISSUE TITLE
problems trying 0.5.3-rc

### ISSUE THREAD

**Issue by davidmaxwaterman** [Polymer/polymer#1117]
I am trying to update project from a hodge-podge of versions of polymer core and paper elements to something orderly. I think 0.8 isn't ready yet and might be more effort, so I looked at the tags on polymer and saw 0.5.3-rc. Presuming 'rc' is release candidate, and noticing that all the core and paper elements also had that tag, I decided to give that a try.

Unfortunately, it still seems to result in a confusing mixture of versions.

If I simply 'mkdir tmp && cd tmp && bower install Polymer/polymer#~0.5.3-rc', I see that it installed 'core-component-page#0.5.2' and also 'webcomponentsjs#0.5.2'. I'm not sure why it installs core-component-page, and I'm not interested in webcomponentsjs at all (chrome is our only target); but I'm not sure why it is installing 0.5.2 rather than 0.5.3-rc.

Is there a way to test 0.5.3-rc versions of everything?
Is there a way to exclude webcomponentsjs?

Should I just stick to 0.5.2?

**Comment by davidmaxwaterman** [comment]
Actually, trying 0.5.2 also seems to result in bower asking for a resolution :

```
Unable to find a suitable version for polymer, please choose one:
    1) polymer#^0.4.0 which resolved to 0.4.2 and is required by core-component-page#0.4.2, core-shared-lib#0.4.2 
    2) polymer#^0.5.0 which resolved to 0.5.2 and is required by core-a11y-keys#0.5.2, core-ajax#0.5.2, core-animation#0.5.2, core-collapse#0.5.2, core-component-page#0.5.2, core-dropdown#0.5.2, core-focusable#0.5.2, core-header-panel#0.5.2, core-icons#0.5.2, core-iconset#0.5.2, core-iconset-svg#0.5.2, core-input#0.5.2, core-label#0.5.2, core-media-query#0.5.2, core-meta#0.5.2, core-overlay#0.5.2, core-range#0.5.2, core-selection#0.5.2, core-selector#0.5.2, core-splitter#0.5.2, core-style#0.5.2, core-transition#0.5.2, paper-button#0.5.2, paper-dialog#0.5.2, paper-dropdown#0.5.2, paper-fab#0.5.2, paper-icon-button#0.5.2, paper-input#0.5.2, paper-item#0.5.2, paper-menu-button#0.5.2, paper-shadow#0.5.2, paper-spinner#0.5.2 
    3) polymer#~0.5.2 which resolved to 0.5.2 and is required by Content PushPrefix the choice with ! to persist it to bower.json

? Answer:: 
```

I wonder how to find out which element is requiring such an old version as indicated in option 1?

**Comment by davidmaxwaterman** [comment]
ah, the problem was :

```
GoogleWebComponents/google-apis
```

which was at 0.3.3 and needed older versions, when I switched to 0.4.3 then it doesn't ask for resolutions. In fact, when I do ``` bower list | grep -v 0.5.2```, I just get all 'the other stuff' and nothing to do with polymer at all, so all good.

Closing.
